packages/utils/src/period/periodExtremes.js

```diff
@@ -22,24 +22,23 @@ export const getMostAncientPeriod = periods => getPeriodExtreme(periods, false);
 
 /**
  * @param {string[]} periods An array of periods
- * @param {boolean} gettingMostRecent Should the function return the most recent period?
+ * @param {boolean} descending Whether to use descending order or not
```

Comment:
I find the old variable name much easier to think about!

Reply:
I was very confused by the old variable name, that' why I changed it... Using a boolean variable may be the problem in the first place, I will try a different approach